aggregate/aggregator.go

```diff
@@ -91,7 +106,7 @@ func (a *Aggregator) AddRecord(partitionKey string, hasPartitionKey bool, data [
 	pkeyFieldSize := protowire.SizeVarint(pKeyIdx) + fieldNumberSize
 	// Total size is byte size of data + pkey field + field number of parent proto
 
-	if a.getSize()+protowire.SizeBytes(dataFieldSize+pkeyFieldSize)+fieldNumberSize+pKeyAddedSize >= maximumRecordSize {
+	if a.getSize()+protowire.SizeBytes(dataFieldSize+pkeyFieldSize)+fieldNumberSize+pKeyAddedSize >= defaultMaximumRecordSize {
```

Comment:
should it be using the `maximumRecordSize` config ?
```suggestion
	if a.getSize()+protowire.SizeBytes(dataFieldSize+pkeyFieldSize)+fieldNumberSize+pKeyAddedSize >= a.maximumRecordSize {
```

Reply:
Oh right. My bad, I missed it.